src/components/MessagesTile/MessagesTile.tsx

```diff
@@ -1,10 +1,13 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import { Tile } from '../tiles/Tile';
 import { HomeStackScreenProps } from '../../navigators';
-import { useUser, useHasNewMessagesFromUsers } from '../../hooks';
+import { useUser } from '../../hooks';
```

Comment:
Same here with the import- we want want to be careful not to import from files like `/hooks` because that includes a bunch of files that aren't relevant to this component.